py/torch_tensorrt/fx/converters/acc_ops_converters.py

```diff
@@ -268,32 +159,36 @@ def acc_ops_conv_transposend(
             input_val.shape[1] != -1
         ), "Channel dim can't be dynamic for transpose convolution."
 
-    # for now we'll assume bias is constant Tensor or None,
-    # and bias being ITensor is not supported in TensorRT api
-    # right now
-    if kwargs["bias"] is not None and not isinstance(kwargs["bias"], torch.Tensor):
-        raise RuntimeError(
-            f"ConvTranspose {name} has bias of type {type(kwargs['bias'])}, Expect Optional[Tensor]"
-        )
-    bias = to_numpy(kwargs["bias"])  # type: ignore[arg-type]
+    if not isinstance(kwargs["bias"], TRTTensor):
```

Comment:
Do this in a separate PR and centralize conv^T

Reply:
Instanced #1996 for this